### PR TITLE
Ignore elements

### PR DESCRIPTION
**Pull request by arunthampi**
In reference to https://github.com/Pomax/react-onclickoutside/issues/9

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -33,6 +33,8 @@
   var registeredComponents = [];
   var handlers = [];
 
+  var IGNORE_CLASS_REGEXP = new RegExp('ignore-react-onclickoutside');
```
Comment (Pomax) [inline index.js]: no need for this to be a regexp, actually, let's just make this `var IGNORE_CLASS = "ignore-react-onclickoutside";`, see below

File: index.js [hunk 2]
```diff
@@ -48,7 +50,7 @@
           // thinking in terms of Dom node nesting, running counter
           // to React's "you shouldn't care about the DOM" philosophy.
           while(source.parentNode) {
-            found = (source === localNode);
+            found = (source === localNode || source.className.match(IGNORE_CLASS_REGEXP));
```
Comment (Pomax) [inline index.js]: classList lets us as targeted questions, so we can do this:
```
source.classList.contains(IGNORE_CLASS)
```
without needing to match regexp on the HTML node "class" node attribute values.

**Comment by arunthampi** [comment]
Thanks for the feedback, this is much nicer -- I've squashed the fixing commit

**Comment by Pomax** [comment]
nice! merging, and I'll spin an updated version

**Pull request merged**